content/shared/influxdb3-plugins/plugins-library/official/basic-transformation.md

```diff
@@ -1,60 +1,70 @@
+|----------------------|--------|------------------|---------------------------------------------------|
+| `measurement`        | string | required         | Source measurement containing data to transform   |
+| `target_measurement` | string | required         | Destination measurement for transformed data      |
+| `target_database`    | string | current database | Database for storing transformed data             |
+| `dry_run`            | string | "false"          | When "true", logs transformations without writing |
 
 ### Transformation parameters
 
-| Parameter | Type | Default | Description |
-|-----------|------|---------|-------------|
-| `names_transformations` | string | none | Field/tag name transformation rules. Format: `'field1:"transform1 transform2".field2:"transform3"'` |
-| `values_transformations` | string | none | Field value transformation rules. Format: `'field1:"transform1".field2:"transform2"'` |
-| `custom_replacements` | string | none | Custom string replacements. Format: `'rule_name:"find=replace"'` |
-| `custom_regex` | string | none | Regex patterns for field matching. Format: `'pattern_name:"temp%"'` |
+| Parameter                | Type   | Default | Description                                                                                         |
+|--------------------------|--------|---------|-----------------------------------------------------------------------------------------------------|
+| `names_transformations`  | string | none    | Field/tag name transformation rules. Format: `'field1:"transform1 transform2".field2:"transform3"'` |
+| `values_transformations` | string | none    | Field value transformation rules. Format: `'field1:"transform1".field2:"transform2"'`               |
+| `custom_replacements`    | string | none    | Custom string replacements. Format: `'rule_name:"find=replace"'`                                    |
+| `custom_regex`           | string | none    | Regex patterns for field matching. Format: `'pattern_name:"temp%"'`                                 |
 
 ### Data selection parameters
 
-| Parameter | Type | Default | Description |
-|-----------|------|---------|-------------|
-| `window` | string | required (scheduled only) | Historical data window. Format: `<number><unit>` (for example, `"30d"`, `"1h"`) |
-| `included_fields` | string | all fields | Dot-separated list of fields to include (for example, `"temp.humidity"`) |
-| `excluded_fields` | string | none | Dot-separated list of fields to exclude |
-| `filters` | string | none | Query filters. Format: `'field:"operator value"'` |
+| Parameter         | Type   | Default                   | Description                                                          |
+|-------------------|--------|---------------------------|----------------------------------------------------------------------|
+| `window`          | string | required (scheduled only) | Historical data window. Format: `<number><unit>` (e.g., "30d", "1h") |
```

Comment:
The script should replace "e.g., " with "for example, ", but we should also replace it in the plugin repo docs permanently.